logstash-core/lib/logstash-core/version.rb

```diff
@@ -5,4 +5,4 @@
 # Note to authors: this should not include dashes because 'gem' barfs if
 # you include a dash in the version string.
 
-LOGSTASH_CORE_VERSION = "6.0.0-alpha1"
+LOGSTASH_CORE_VERSION = "6.0.0.alpha1"
```

Comment:
Crap, you are right, I thought we were using the yaml in the `rakelib/artifacts.rake` but we use the constant.